iree/tools/BUILD

```diff
@@ -77,6 +77,7 @@ cc_binary(
 
 cc_library(
     name = "iree_opt_library",
```

Comment:
It feels a bit weird for the name to still be "iree_opt_library" if it includes a main. Can we call it "iree_opt_main" (similar to gtest_main)?

Reply:
Hm, it's the same for `iree_translate_library`. Yeah, I agree it would be nice to chance this. Will do so in a follow up PR if that"s acceptable, also touching `iree_translate_library`.

Reply:
SGTM